internal/budgetadjustments/flags.go

```diff
@@ -0,0 +1,96 @@
+package budgetadjustments
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/nobl9/sloctl/internal/csv"
+)
+
+const (
+	flagAdjustment = "adjustment-name"
+	flagFrom       = "from"
+	flagTo         = "to"
+	flagProject    = "project"
```

Comment:
`slo-project`? project is related to SLO not to adjustment so it could be confusing.